<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidChassisTypeSelect</class>
 <widget class="QWidget" name="GuidChassisTypeSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>moveMode：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="customComboBox" name="comboBox_moveStyle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>chassisType：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="customComboBox" name="comboBox_funcType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please obtain the following robot parameters for configuration according to the product specification or design drawings. Click next after completion.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
     <item>
      <widget class="QGroupBox" name="groupBox_image">
       <property name="title">
        <string>Picture</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_image">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_chassisParameter">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>14</number>
          </property>
          <item>
           <widget class="QLabel" name="label_wheelRadius">
            <property name="text">
             <string>wheelRadius:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Aaxis">
            <property name="text">
             <string>A axis:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Baxis">
            <property name="text">
             <string>B axis:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_theta">
            <property name="text">
             <string>theta:[°]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_steerAngelRange">
            <property name="text">
             <string>minAngle:[°]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_steerAngelRange_max">
            <property name="text">
             <string>maxAngle:[°]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_encoderRange">
            <property name="text">
             <string>minAbsEncoder:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_encoderRange_max">
            <property name="text">
             <string>maxAbsEncoder:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_EncoderType">
            <property name="text">
             <string>encoderType:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_wheelRadius">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_Aaxis">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_Baxis">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_theta">
            <property name="toolTip">
             <string>range[0,120]</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>120.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_steerMinAngel">
            <property name="toolTip">
             <string>range[-90,maxAngle]</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_steerMaxAngel">
            <property name="toolTip">
             <string>range[minAngle，90]</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_encoderMin">
            <property name="toolTip">
             <string>range[0,maxAbsEncoder]</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1800.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_encoderMax">
            <property name="toolTip">
             <string>range[minAbsEncoder，1800]</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1800.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customComboBox" name="comboBox_encoderType"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
